### PR TITLE
[X11] Add more details to large icon size warning

### PR DESCRIPTION
**Pull request by theromis**
Searched my icons till not add this size info, I think it will simplify life for a lot of developers/designers.

### CODE REVIEW

**Comment by AThousandShips** [comment]
Please make your commit message more clear, the title of the PR would be good

File: platform/linuxbsd/x11/display_server_x11.cpp [hunk 1]
```diff
@@ -5212,7 +5212,7 @@ void DisplayServerX11::set_icon(const Ref<Image> &p_icon) {
 			if (g_set_icon_error) {
 				g_set_icon_error = false;
 
-				WARN_PRINT("Icon too large, attempting to resize icon.");
+				WARN_PRINT(vformat("Icon too large, attempting to resize icon (%dx%d).", w, h));
```
Comment (Calinou) [inline platform/linuxbsd/x11/display_server_x11.cpp:5215]:
```suggestion
				WARN_PRINT(vformat("Icon too large (%dx%d), attempting to downscale icon.", w, h));
```

**Comment by theromis** [comment]
should I currently do any changes?

**Comment by theromis** [comment]
@AThousandShips should I do any corrections?

**Comment by AThousandShips** [comment]
You need to fix the commit message and please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Comment by theromis** [comment]
@AThousandShips updated please check

**Comment by theromis** [comment]
@AThousandShips is it possible to merge and close this PR?

**Review by AThousandShips — APPROVED** [review]
LGTM

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**